internal/command/ping_pong/ping_pong_command.go

```diff
@@ -0,0 +1,18 @@
+package ping_pong
```

Comment:
```suggestion
package pingpong
```
パッケージ名に `_` はあまり入れないでほしい
あわせてディレクトリ名も直してね